### DISCUSSION TITLE
Deleting tag with the delete GUI control

### DISCUSSION THREAD

**Discussion by viharm** [firefly-iii/firefly-iii#5544] [Firefly III]
I'm using 5.6.10, and encountered an interesting issue when clicking on the GUI control for clearing the tags (shown by the trashcan icon at the right end of the field.

If I use this GUI control to clear the tags, and update the transaction, the tags stay. If I manually delete the tag by using backspace or by clicking the small "x" next to the tag name, then it gets deleted.

Is anyone else having this issue? Or is it my setup?

**Comment by JC5** [comment]
That's probably a bug, I'll pick it up for the new layout.
